server/utils/index.js

```diff
@@ -32,7 +32,6 @@ export const stringifyWithCheck = message => {
     if (message.data) {
       return stringifyWithCheck(message.data);
     } else {
-      console.log(message);
       return `unable to unfurl message: ${message}`;
```

Comment:
this console.log is fine since I would like to print what was the message that we couldn't unfurl